preview/index.ts

```diff
@@ -14,68 +18,62 @@ const PREVIEW_DATA_SOURCE_FILE_PATH = './preview/sampleData.csv';
 
 const logFeedback = (
 	versionNumber: string,
-	newsletters: EmailNewsletter[],
+	newsletters: NewsletterResponse[],
 ): void => {
 	console.log({ versionNumber });
-	console.log('INDEX\tVALID\tCurrent\t CANCELLED\tNAME');
+	console.log('INDEX\tVALID\tCANCELLED\tNAME');
 	newsletters.forEach((newsletter, index) => {
 		console.log(
 			index,
 			'\t',
-			EmailNewsletterType.is(newsletter) ||
-				CancelledEmailNewsletterType.is(newsletter),
-			'\t',
-			EmailNewsletterType.is(newsletter),
+			NewsletterResponseCodec.is(newsletter),
 			'\t',
 			CancelledEmailNewsletterType.is(newsletter),
```

Comment:
as above - probably best to remove `NewsletterResponseCodec.is(newsletter),` since its always true